content/posts/fpcast.md

```diff
@@ -0,0 +1,478 @@
+---
+title: "Pyodide Function Pointer Cast Removal"
+date: 2021-12-06T20:27:27-08:00
+draft: true
+tags: ["internals"]
+author: "Hood Chatham"
+# author: ["Me", "You"] # multiple authors
+showToc: true
+TocOpen: false
+draft: false
+hidemeta: false
+comments: false
+description: "Why old versions of Pyodide had a low recursion limit and how version 0.19.0 supports a much higher one."
+# canonicalURL: "https://canonical.url/to/page"
+disableHLJS: true # to disable highlightjs
+disableShare: false
+hideSummary: false
+searchHidden: true
+ShowReadingTime: true
+ShowBreadCrumbs: true
+ShowPostNavLinks: true
+cover:
+    image: "<image path/url>" # image path/url
+    alt: "<alt text>" # alt text
+    caption: "<text>" # display caption under cover
```

Comment:
To be safe we should probably keep those 3 lines commented, even with hidden=True